### DISCUSSION TITLE
List of translators for one project/component

### DISCUSSION THREAD

**Discussion by buhtz** [WeblateOrg/weblate#9439] [Q&A]
I do use the Weblate instance at translate.coderberg.org

I can see translators per String or language. But I would like to see a list of all known translators for one component.

    https://translate.codeberg.org/projects/backintime/common/#information

Is it possible? Did I missed something in the front end?

**Comment by nijel** [comment]
You can use the reporting tool to get a list of contributors, see https://docs.weblate.org/en/latest/devel/reporting.html.

In Weblate 4.18, the list of recent translators is also reachable by clicking on the number of translators on the info page you mention. So, in this case, your server is missing an upgrade ;-).
